Packs/MandiantAdvantageThreatIntelligence/Integrations/ThreatIntelligence/README.md

```diff
@@ -0,0 +1,222 @@
+Integrate with Mandiant Advantage
+This integration was integrated and tested with version xx of Mandiant Advantage Threat Intelligence
+
```

Comment:
Please go over these lines and give here a short explanation about the integration with Mandiant
https://xsoar.pan.dev/docs/documentation/readme_file